main.go

```diff
@@ -109,6 +110,7 @@ func init() {
 	_ = statusv1beta1.AddToScheme(scheme)
 	_ = mutationsv1alpha1.AddToScheme(scheme)
 	// +kubebuilder:scaffold:scheme
+	flag.Var(disabledBuiltin, "disable-builtin", "disable opa built-in function, this flag can be declared more than once.")
```

Comment:
nit: `disabledBuiltins` plural for the variable name, as it's a slice. The flag name is fine.

Reply:
Any guidance we can provide on what valid builtins are for this flag? Can we link to OPA documentation for example?